doc/guides/bluetooth/overview.rst

```diff
@@ -0,0 +1,88 @@
+.. _bluetooth-overview:
+
+Overview
+########
+
+.. contents::
+    :local:
+    :depth: 2
+
+Supported Features
+******************
+
+Zephyr comes integrated with a feature-rich and highly configurable
+Bluetooth stack.
+
+* Bluetooth 5.0 compliant (ESR10)
+
+  * Bluetooth subsystem and infrastructure
+
+  * Highly configurable
+
+      * Features, buffer sizes/counts, stack sizes, etc.
+  
+  * Portable to all architectures supported by Zephyr (including big and
+    little endian, alignment flavors and more)
+
+  * Support for all combinations of Host and Controller builds:
+
+    * Controller-only (HCI) over UART, SPI and USB physical transports
+    * Host-only over UART, SPI and IPM (shared memory)
+    * Combined (Host + Controller)
+
+* Bluetooth Low Energy Controller support (LE Link Layer)
+
+  * Unlimited role and connection count, all roles supported
+  * Concurrent multi-protocol support ready
+  * Intelligent scheduling of roles to minimize overlap
+  * Portable design to any open BLE radio, currently supports Nordic
```

Comment:
This part will only be true after 1.14, so maybe we should not advertise it yet, or?

Reply:
It's true in master, this doc is in master so I think it's fine